### PR TITLE
Adding basic config object and reorganize file interface.

### PR DESCRIPTION
**Pull request by heistermann**
This PR demonstrates how a config object is initialised and passed to all modules. In the modules, the config object can be modified. The config object can be saved (as a kind of project file), and initialised from file.

I added the "File" menue bar to avoid overcrowding the Properties widget with buttons.

### CODE REVIEW

File: wradvis/config.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from configparser import ConfigParser
+import os
+
+# Initialise default configuration object
+
+def init_conf():
+
+    conf = ConfigParser()
+
+    conf["dirs"] = {"data": os.path.join(os.getcwd(), "data\\rw\\20160529") }
```
Comment (kmuehlbauer) [inline wradvis/config.py]: Is there a way to have normal slashes?  -> `data/rw/20160529`

The backslash stuff won't work on linux, but slashes should also work on windows.

**Comment by kmuehlbauer** [comment]
One minor comment, otherwise LGTM. 

We should also discuss the whole layout of the gui in another issue. See #13

**Comment by kmuehlbauer** [comment]
What about the `self.sourcebox` and the `dirLabel`. This isn't shown atm.

**Comment by heistermann** [comment]
You are right, I removed it. But it would be good to show the current data directory in the properties widget, I agree. Question is then: should it be manipulated by a button (as before), or by the `File` menu (as here).

**Comment by kmuehlbauer** [comment]
The Menu makes much more sense, than the single button. So I would favour this.

**Comment by kmuehlbauer** [comment]
You would need to add `mainMenu` to `vbox` as first widget. Otherwise the layout breaks.

**Comment by kmuehlbauer** [comment]
... just because it looks better. YMMD! :tada:

**Comment by kmuehlbauer** [comment]
Is this good to go? LGTM.

**Pull request merged**